core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -31,7 +31,7 @@
   LOGS_DIR(Name.LOGS_DIR, String.format("${%s}/logs", Name.WORK_DIR)),
   METRICS_CONF_FILE(Name.METRICS_CONF_FILE,
       String.format("${%s}/metrics.properties", Name.CONF_DIR)),
-  NETWORK_HOST_RESOLUTION_TIMEOUT_MS(Name.NETWORK_HOST_RESOLUTION_TIMEOUT_MS, 5000),
+  NETWORK_HOST_RESOLUTION_TIMEOUT_MS(Name.NETWORK_HOST_RESOLUTION_TIMEOUT_MS, "5S"),
```

Comment:
can we use lower case here?
reason being constant like `1M` is typically read as `1MB` as opposed to `1 min`.